conda_recipes/maya-vray-2025/README.md

```diff
@@ -0,0 +1,22 @@
+# V-Ray conda build recipe
+Use this recipe to create a V-Ray for Maya conda package to use with AWS Deadline Cloud. Conda packages let you customize the software you can use with your Deadline Cloud deployment. Read more about how to host a conda channel for these custom conda packages below.
+
+## Download the archive file
+- Download the `vray_adv_62002_maya2025_rhel8` full download file from [Chaos](https://download.chaos.com/downloads/23688/vray-maya-2025-62002-adv)\
+**_NOTE:_** Need to have a Chaos account to access the link
+
+## Build Conda Package
+- place it in the `conda_recipes/archive_files` directory in your git clone of the
```

Comment:
Since this is a new section, it's a new context that someone can jump to and read independently. Instead of "it", say something like "the `vray_adv_62002_maya2025_rhel8` file" to be explicit. It might also make sense to include this in the `## Download the archive file` section instead of here.